### DISCUSSION TITLE
Unable to upgrade inventree

### DISCUSSION THREAD

**Discussion by rohitsingal1** [inventree/InvenTree#6372] [Q&A]
While updating inventree getting the following error. 
My Python version is Python 3.10.12.

> sudo apt upgrade inventree
> Reading package lists... Done
> Building dependency tree... Done
> Reading state information... Done
> Calculating upgrade... Done
> Some packages could not be installed. This may mean that you have
> requested an impossible situation or if you are using the unstable
> distribution that some required packages have not yet been created
> or been moved out of Incoming.
> The following information may help to resolve the situation:
> 
> The following packages have unmet dependencies:
>  inventree : Depends: python3.9 but it is not installable
>              Depends: python3.9-venv but it is not installable
>              Depends: python3.9-dev but it is not installable
> E: Broken packages

**Comment by eeintech** [comment]
Related to: https://github.com/inventree/InvenTree/issues/6154
